crates/sui-indexer-alt/src/models/objects.rs

```diff
@@ -145,3 +148,42 @@ pub struct StoredObjInfo {
     pub name: Option<String>,
     pub instantiation: Option<Vec<u8>>,
 }
+
+pub fn create_stored_obj_info(
+    object_id: ObjectID,
```

Comment:
Is this value ever not the same as `object.id()`? Should we get rid of the parameter?